src/main/java/gameframework/drawing/GameCanvas.java

```diff
@@ -27,11 +27,18 @@
 	void setSize(int width, int height);
 
 	void addTo(Frame frame);
+	
+	public InputManager getInputManager();
 
+	/* There is now an InputManager class that handles the listeners for the canvas. You must use it instead of the following 
```

Comment:
please tell the developer what he should write instead of his deprecated code. Don't let him search for information.